src/client/vue.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   css: {
     loaderOptions: {
       sass: {
-        prependData: `@import "@/core/styles/_variables.scss";`,
+        additionalData: `@import "@/core/styles/_variables.scss";`,
```

Comment:
`prependData` has been changed to `additionalData` from `sass-loader` v8 onwards
https://github.com/webpack-contrib/sass-loader/issues/865